tools/integration_tests/mount_timeout/gcsfuse_mount_relaxed_timeout_test.go

```diff
@@ -0,0 +1,237 @@
+// Copyright 2024 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package mount_timeout
+
+import (
+	"fmt"
+	"os"
+	"path"
+	"testing"
+	"time"
+
+	"github.com/googlecloudplatform/gcsfuse/v2/cfg"
+	"github.com/googlecloudplatform/gcsfuse/v2/tools/integration_tests/util/mounting"
+	"github.com/googlecloudplatform/gcsfuse/v2/tools/integration_tests/util/setup"
+	"github.com/googlecloudplatform/gcsfuse/v2/tools/util"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/suite"
+)
+
+func TestMountTimeoutWithRelaxedThreshold(t *testing.T) {
+	if os.Getenv("TEST_ENV") == testEnvGCENonUSCentral {
+		suite.Run(t, new(MountTimeoutTestWithRelaxedThreshold))
+	} else {
+		t.Skip()
+	}
+}
+
+type MountTimeoutTestWithRelaxedThreshold struct {
```

Comment:
this and common-timeout test suite are almost duplicate other than the value of timeout. Can we reuse this test suite for common-relaxed timeout test?